tests/data/dynamic_driver/mesh_1.dummy

```diff
@@ -0,0 +1,13 @@
+dummy_mesh_1
```

Comment:
can you use "qgismesh" or "qgismemory" or "qgsm" or something like that instead of dummy all over the place
? thanks